cinema/serializers.py

```diff
@@ -68,13 +80,60 @@
             "movie_title",
             "cinema_hall_name",
             "cinema_hall_capacity",
+            "tickets_available",
         )
 
 
+class TicketSeatRowSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Ticket
+        fields = ("row", "seat")
+        read_only_fields = fields
```

Comment:
The `read_only_fields` attribute is set to `fields`, which is not correct. It should be a tuple or list of field names that are read-only. Consider changing it to `read_only_fields = ('row', 'seat')`.